src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public static String getRandomColor() {
+        int index = new Random().nextInt(Ball.Color.values().length);
```

Comment:
Creating a new Random instance every time the method is called is inefficient and unnecessary. You should create a single Random instance as a class field and reuse it.